AUTHORS

```diff
@@ -164,6 +164,7 @@ janost <janost@tuta.io>
 Jaroslav Lichtblau <svetlemodry@users.noreply.github.com>
 Jaroslav Malec (dzarda) <dzardacz@gmail.com>
 jaseg <githubaccount@jaseg.net>
+Jaspitta <ste.scarpitta@gmail.com>
```

Comment:
Please remove this change from the PR. You will be added automatically to the file once the PR has been merged.